services/managers/fleetup_manager.py

```diff
@@ -0,0 +1,118 @@
+                           "type":row["Type"],
+                           "timer_type":row["TimerType"],
+                           "expires": (datetime.strptime(row["ExpiresString"], "%Y-%m-%d %H:%M:%S")),
+                           "notes":row["Notes"]} for row in ftimers["Data"]}
+
+    @staticmethod
+    def get_fleetup_doctrines():
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/Doctrines/" + str(groupid) + ""
+        jsondata = requests.get(url).content
+        fdoctrines=json.loads(jsondata.decode())
+
+        return {"fleetup_doctrines":fdoctrines["Data"]}
+
+    @staticmethod
+    def get_fleetup_doctrine(doctrinenumber):
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/DoctrineFittings/%s" % doctrinenumber
+        jsondata = requests.get(url).content
+        fdoctrine=json.loads(jsondata.decode())
+
+        return {"fitting_doctrine":fdoctrine}
+
+    @staticmethod
+    def get_fleetup_fittings():
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/Fittings/" + str(groupid) + ""
+        jsondata = requests.get(url).content
+        ffittings=json.loads(jsondata.decode())
+
+        return {row["FittingId"]:{"fitting_id":row["FittingId"],
+                               "name":row["Name"],
+                               "icon_id":row["EveTypeId"],
+                               "hull":row["HullType"],
+                               "shiptype":row["ShipType"],
+                               "estimated":row["EstPrice"],
+                               "faction":row["Faction"],
+                               "categories":row["Categories"],
+                               "last_update":(datetime.strptime(row["LastUpdatedString"], "%Y-%m-%d %H:%M:%S"))} for row in ffittings["Data"]}
+
+    @staticmethod
+    def get_fleetup_fitting(fittingnumber):
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/Fitting/%s" % fittingnumber
+        jsondata = requests.get(url).content
+        ffitting=json.loads(jsondata.decode())
+
+        return {"fitting_data":ffitting["Data"]}
+
+    @staticmethod
+    def get_fleetup_doctrineid(fittingnumber):
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/Fitting/%s" % fittingnumber
+        jsondata = requests.get(url).content
+        fdoctrineid=json.loads(jsondata.decode())
+
+        return fdoctrineid['Data']['Doctrines'][0]['DoctrineId']
+
+    @staticmethod
+    def get_fleetup_fitting_eft(fittingnumber):
+        url = "http://api.fleet-up.com/Api.svc/" + str(appkey) + "/" + str(userid) + "/" + str(apiid) + "/Fitting/%s/eft" % fittingnumber
+        jsondata = requests.get(url).content
+        ffittingeft=json.loads(jsondata.decode())
+
+        return {"fitting_eft":ffittingeft["Data"]["FittingData"]}
```

Comment:
.a good example

http://stackoverflow.com/questions/24210792/checking-for-timeout-error-in-python